frontend-sdk/README.md

```diff
@@ -0,0 +1,151 @@
+# hanko-frontend-sdk
+
+This package utilizes the [Hanko API](https://github.com/teamhanko/hanko/blob/main/backend/README.md) to provide
+functionality that allows an easier UI integration. It is meant for use in browsers only.
+
+## Installation
+
+```shell
+# npm
+npm install @teamhanko/hanko-frontend-sdk
+
+# yarn
+yarn add @teamhanko/hanko-frontend-sdk
+
+# pnpm
+pnpm install @teamhanko/hanko-frontend-sdk
+```
+
+## Usage
+
+Import as a module:
+
+```typescript
+import { Hanko } from "@teamhanko/hanko-frontend-sdk"
+
+const hanko = new Hanko("http://localhost:3000")
+```
+
+With a script tag via CDN:
+
+```html
+<script src="https://unpkg.com/@teamhanko/hanko-frontend-sdk/dist/sdk.umd.js"></script>
+
+<script>
+    const hanko = new hankoFrontendSdk.Hanko("http://localhost:3000")
+    ...
+</script>
+```
+
+## Documentation
+
+To see the latest documentation, please click [here](https://teamhanko.github.io/hanko/jsdoc/@teamhanko/hanko-frontend-sdk/0.0.1-alpha).
+
+## Exports
+
+### SDK
+
+- `Hanko` - A class that bundles all functionalities.
+
+### Clients
+
+- `ConfigClient` - A class to fetch configurations.
+- `UserClient` - A class to manage users.
+- `WebauthnClient` - A class to handle WebAuthN-related functionalities.
```

Comment:
"WebAuthn" is the official spelling

Reply:
Multiple occasions in this readme

Reply:
Thanks! Changed it across all files and re-generated the docs.